content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -807,6 +807,16 @@ These are some requirements and semantics of the API:
   - `Count`: use to indicate that the Pod should be handled in the default way.
      The counter towards the `.spec.backoffLimit` should be incremented.
 
+{{< note >}}
+Since the 1.27 version Kubelet transitions deleted pods to a terminal phase
```

Comment:
Shouldn't this go in the documentation for Pod disruptions?

We didn't change the behavior of the job controller, did we? But maybe we can have a note that the job controller only considers for disruption policies the pods that are in Failed phase.

Reply:
Ah, that's the one.

LGTM from sig apps

Reply:
Done